### PR TITLE
fix plugin documentation with current example

### PR DESCRIPTION
**Pull request by opyate**
The portions of [0.13.1 Plugin documentation](http://www.scala-sbt.org/0.13.1/docs/Extending/Plugins.html) that I removed in my commit doesn't work for me, and I have a reproducible test here:   https://gist.github.com/opyate/7943389

### CODE REVIEW

File: src/sphinx/Extending/Plugins.rst [hunk 1]
```diff
@@ -95,110 +95,39 @@ By Example
...
 Edit `project/plugins.sbt` to contain:
 
 ::
 
-    libraryDependencies += "org.clapper" %% "grizzled-scala" % "1.0.4"
+    addSbtPlugin("com.earldouglas" % "xsbt-web-plugin" % "0.5.0")
 
 If sbt is running, do `reload`.
 
-1c) Automatically managed: command line approach
```
Comment (jsuereth) [inline src/sphinx/Extending/Plugins.rst:120]: This is still a valid mechanism to configure plugins.  Do you really think we should drop it?

File: src/sphinx/Extending/Plugins.rst [hunk 2]
```diff
@@ -95,110 +95,39 @@ By Example
...
-    ...
-
-To switch back to the main project:
-
-.. code-block:: console
-
-    > reload return
-    [info] Set current project to root (in build file:/Users/harrah/demo2/)
-
-1d) Project dependency
```
Comment (jsuereth) [inline src/sphinx/Extending/Plugins.rst:153]: I also use this a bit, so I'd like to keep this doc around.

**Comment by jsuereth** [comment]
Very much appreciate the pull request to improve docs.  IF you leave the sections you've removed around, I'll update them with more up-to-date information.

Thanks!

File: src/sphinx/Extending/Plugins.rst (hunk 1 above)
Comment (opyate) [inline src/sphinx/Extending/Plugins.rst:120]: Neither `set libraryDependencies += "com.earldouglas" % "xsbt-web-plugin" % "0.5.0"` nor `addSbtPlugin("com.earldouglas" % "xsbt-web-plugin" % "0.5.0")` works from the CLI for me (even after `reload plugins`).

Also, not only does `libraryDependencies += "com.earldouglas" % "xsbt-web-plugin" % "0.5.0"` not work from the CLI -- it doesn't work from project/myplugin.sbt.

I tried all this on a clean Linux install, as per https://gist.github.com/opyate/7943389. I.e. no ~/.sbt, no .sbtopts, etc.

Please provide an example here and I'll incorporate it into the pull request.

Comment (jsuereth) [inline src/sphinx/Extending/Plugins.rst:120]: That would be 

```
reload plugins

set addSbtPlugin("com.earldouglas" % "xsbt-web-plugin" % "0.5.0")

session save

reload return
```

**Comment by eed3si9n** [comment]
I'm closing this due to inactivity

**Pull request closed without merging**